app/views/users/index.html.erb

```diff
@@ -1,19 +1,20 @@
 <main>
-  <section>
+  <section class="users-container d-flex justify-content-center flex-column align-items-center">
     <% if @users.empty? %>
       <h2>There are no users in the database</h2>
     <% else %>
       <h2>Take a look at all of the users and their number of posts</h2>
 
       <% @users.each do |user| %>
-        <article class="users-section">
-          <div class="users-section-card">
+        <article class="users-section d-flex justify-content-center align-items-center">
+            <%= image_tag "user-regular.svg" %>
+
             <%= link_to user_path(user.id) do %>
-              <img src="<%= user.photo %>" alt="The representation of the user">
-              <h4><%= user.name %></h4>
-              <p>Number of Posts: <%= user.posts_counter %></p>
+              <div class="users-section-card d-flex">
+                <h4><%= user.name %></h4>
```

Comment:
- [ ] I tried to navigate to the post page of the post I created but the posts/show throw the following error:

<img width="899" alt="image" src="https://user-images.githubusercontent.com/77190203/194721343-c891f291-bce9-4936-9a16-442d897c030b.png">

Kindly fix the error, and ensure that the user is able to create comments and add likes.